### PR TITLE
Add template choices to DFRPG class templates

### PR DESCRIPTION
**Pull request by dripton**

### CODE REVIEW

File: Library/Dungeon Fantasy RPG/Classes/Barbarian.gct [hunk 1]
```diff
@@ -2065,6 +2117,15 @@
...
 								},
+								{
+									"id": "15608554-8633-4b15-8e27-a821ccbd640b",
+									"type": "trait_container",
+									"open": true,
+									"name": "Trait Container",
+									"calc": {
+										"points": 0
+									}
+								},
```
Comment (richardwilkes) [inline Library/Dungeon Fantasy RPG/Classes/Barbarian.gct:2128]: What's the point of this empty trait container?

**Comment by richardwilkes** [comment]
Since there is currently no way for users to select the appropriate modifiers for things like Appearance in the template picker, it would likely be better if you created separate instances of each applicable base state (in Barbarian case, it looks like you already narrowed the modifier choices down to two, so create two entries instead, each with just the one modifier enabled). This will cause the trait to be priced appropriately, allowing them to actually meet the criteria.

File: Library/Dungeon Fantasy RPG/Classes/Barbarian.gct (hunk 1 above)
Comment (dripton) [reply Library/Dungeon Fantasy RPG/Classes/Barbarian.gct]: Typo, fixed

**Comment by dripton** [comment]
I agree that would be better.  So far, I have just made minimal changes, finding places where there was a choice and adding the new choice templates.  I can take another pass and try to find places where a trait should be fragmented into multiple traits.

**Comment by dripton** [comment]
I split 0-point Wealth, Appearance, and Languages into fixed point versions.  One issue is that Languages could be taken more than once; I don't see a good way to represent that in the current system.  (I could do Language 1, Language 2, etc.  But how many is a good limit?)

**Comment by richardwilkes** [comment]
Sense of Duty looks like another one that should be expanded to the two possibilities listed (in the Barbarian template, anyway).

**Comment by dripton** [comment]
I split up the Barbarian Sense of Duty.  Looks like other classes with two kinds of Sense of Duty already had them split.

**Comment by richardwilkes** [comment]
I see a few other things that could be split out, but I think I'll just quit nit-picking at merge. This can always be followed up with refinements.

**Pull request merged**